src/styles/main.scss

```diff
@@ -10,5 +10,45 @@ body {
   box-sizing: border-box;
 }
 
+$cell-size: 200px;
+
 @import "_header.scss";
-@import "_catalog.scss";
+
+.catalog {
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+}
+
+@media (max-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
```

Comment:
you could make it by default value in catalog and delete this media

Reply:
updated